### PR TITLE
Caroussel

### PR DESCRIPTION
**Pull request by Sarahshr**
Add Trend musics' carousel

### CODE REVIEW

**Review by Fx-Colliot — DISMISSED** [review]
perfect

**Comment by ThomasMlh** [comment]
It's ok for me ! 👍

**Review by ThomasMlh — DISMISSED** [review]
It's ok for me ! 👍

**Review by ThomasMlh — DISMISSED** [review]
Good 👍

**Review by Fx-Colliot — DISMISSED** [review]
ok

**Review by Fx-Colliot — DISMISSED** [review]
good for me

File: src/App.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { Switch, Route } from "react-router-dom";
 import List from "./components/List";
 import Music from "./components/Music";
 import Nav from "./components/Nav";
-import Home from "./components/Home";
+import Home from "./components/homepage/Home";
+//import TrendingMusic from "./components/TrendingMusic";
```
Comment (LoicBrassart) [inline src/App.js]: Dead code !

**Review by LoicBrassart — CHANGES_REQUESTED** [review]

**Review by LoicBrassart — APPROVED** [review]
NmE !

**Review by ThomasMlh — APPROVED** [review]
It's ok !

**Pull request merged**